website/docs/cloud-docs/no-code-provisioning/provisioning.mdx

```diff
@@ -54,4 +54,11 @@ To launch the no-code workflow:
 
 No-code workspaces have a limited feature set because you cannot access the resource configuration. However, you can edit workspace variables and settings, including notifications, permissions, and run triggers. You can use run triggers to connect the workspace to one or more source workspaces, start new runs when you change workspace variables, or queue destroy runs.
 
-If the module’s configuration changes, you must destroy the workspace and create a new one to provision resources with the updated configuration.
+### Updating Variables
+In order to change variable inputs after provisioning, go to the Variables section in your workspace. There you will see the variables that have been applied. 
+1. Select **Edit** for the variables you want to update.
+2. Enter the desired value
+3. **Plan** and **Apply**.
```

Comment:
This is important to say, but it isn't a variable editing step, so I suggest moving this sentence below the current list:
```suggestion

Start a new run in your workspace to update your existing infrastructure with your new variable value.
```